Problem3.cpp

```diff
@@ -0,0 +1,48 @@
+#include "pch.h"
+#include <gtest/gtest.h>
+
+using namespace std;
+
+string lexicographicallyMinimalString(string, string);
+
+TEST(Lexicographically_Minimal_String, Test1) {
+	EXPECT_EQ(lexicographicallyMinimalString("ACA", "BCF"), "ABCACF");
+}
+
+TEST(Lexicographically_Minimal_String, Test2) {
+	EXPECT_EQ(lexicographicallyMinimalString("acaccczzxxrcv", "bcaccczzxxrca"), "abcacacccccczzxxrcazzxxrcv");
+}
+
+TEST(Lexicographically_Minimal_String, Test3) {
+	EXPECT_EQ(lexicographicallyMinimalString("adfrdsfacascfa", "aszczasr"), "aadfrdsfacascfaszczasr");
+}
+
+string lexicographicallyMinimalString(string j, string s) {
+	string result = "";
+	int l1 = j.length();
```

Comment:
Same as problem #1, I think it's better to use more descriptive names for the variables.

Reply:
It's hard to quickly read the code below because of the var names

Reply:
Ok, I can try to change variables names with descriptive names.